exercises/matthew_bowles/ex02_oo_basics/inc/Circle.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+#include "Shape.h"
+#include <math.h>
```

Comment:
Yes, but nothing in the header file relies on `math.h`. You should only include it in the `.cpp` file in this case.